auacm/angular/src/app/view-scoreboard/view-scoreboard.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="body margin-top">
+
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <div class="row">
+        <div class="col-md-4 pull-left">
+          <h1 class="panel-title pull-left">{{ competition.name }}</h1>
+          <div *ngIf="userData.isAdmin && !ended" class="col-md-4 dropdown">
+            <a class="muted-link dropdown-toggle hover-pointer" type="button"
+                id="#dropdownMenu1" data-toggle="dropdown" aria-haspopup="true"
+                aria-expanded="true">
+              Manage
+              <span class="caret"></span>
+            </a>
+            <ul id="dropdownMenu1" class="dropdown-menu">
+              <li><a [routerLink]="['/competition/' + competition.cid + '/edit']">Edit</a></li>
+              <li><a [routerLink]="['/competition/' + competition.cid + '/teams']">Teams</a></li>
+            </ul>
+          </div>
+        </div>
+        <!-- Countdown to competition start -->
+        <div class="col-md-4 row" *ngIf="timeUntil > 0">
+          <div class="text-center">
+            <h3 class="panel-title">Time Until Start: {{ timeUntil | secondsToHours }}</h3>
+          </div>
+        </div>
+        <h2 class="panel-title col-md-4 pull-right text-right"
+            *ngIf="active || !ended">
```

Comment:
I have no clue. I just copied it over from the old project. Let me know what you want me to do with it.